receiver/mysqlreceiver/metadata.yaml

```diff
@@ -279,3 +283,13 @@ metrics:
       monotonic: false
       aggregation: cumulative
     attributes: [threads]
+  mysql.created_tmp:
+    enabled: true
+    description: The number of created temporary resources.
+    unit: 1
+    sum:
+      value_type: int
+      input_type: string
+      monotonic: true
+      aggregation: cumulative
+    attributes: [tmp_resource]
```

Comment:
What do you think about calling the metric `mysql.tmp_resources`, and clarifying in the description that these are counts of tmp resources that were created (though probably this is obvious)?

Reply:
Only those are the tmp related metrics, so this is fine :+1:

Reply:
hmm, there is also `Slave_open_temp_tables`, but still `mysql.tmp_resources` sounds good

Reply:
done